app/views.py

```diff
@@ -1,4 +1,4 @@
-from flask.ext.httpauth import HTTPBasicAuth
+from flask_httpauth import HTTPBasicAuth
```

Comment:
Yes, there is. Python 3 warns firstly that it's deprecated, then it gives an error.